### PR TITLE
fix(skeleton): [skeleton]add skeleton of smb theme

### PR DESCRIPTION
**Pull request by James-9696**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/theme/src/skeleton/index.less [hunk 1]
```diff
@@ -28,7 +28,12 @@
 
   &-item__title {
     width: var(--ti-skeleton-title-width);
-    margin-bottom: var(--ti-skeleton-title-margin-bottom);
+    height: var(--ti-skeleton-title-height);
+  }
+
+  &-item__rows {
+    margin-top: var(--ti-skeleton-row-margin-top);
+    margin-bottom: var(--ti-skeleton-rows-margin-bottom);
```
Comment (kagol) [inline packages/theme/src/skeleton/index.less:36]: 建议统一命名规则，不要一个row：`--ti-skeleton-row-margin-top`，一个rows：`--ti-skeleton-rows-margin-bottom`

File: packages/theme/src/skeleton-item/vars.less [hunk 2]
```diff
@@ -13,4 +13,6 @@
   --ti-skeleton-item-image-medium-size-width: var(--ti-common-size-25x, 100px);
```
Comment (Youyou-smiles) [inline packages/theme/src/skeleton-item/vars.less]: 将var定义的100px可以去掉。

File: packages/theme/src/skeleton-item/vars.less [hunk 3]
```diff
@@ -13,4 +13,6 @@
...
   --ti-skeleton-item-image-large-size-height: var(--ti-common-size-50x, 200px);
   --ti-skeleton-item-image-large-size-width: var(--ti-common-size-50x, 200px);
```
Comment (Youyou-smiles) [inline packages/theme/src/skeleton-item/vars.less]: 添加定义变量注释

**Review by Youyou-smiles — APPROVED** [review]

File: packages/theme/src/skeleton/index.less (hunk 1 above)
Comment (James-9696) [reply packages/theme/src/skeleton/index.less:36]: 已修改

File: packages/theme/src/skeleton-item/vars.less (hunk 3 above)
Comment (James-9696) [reply packages/theme/src/skeleton-item/vars.less]: 已修改

File: packages/theme/src/skeleton-item/vars.less (hunk 2 above)
Comment (James-9696) [reply packages/theme/src/skeleton-item/vars.less]: 已修改

File: packages/theme/src/skeleton-item/smb-theme.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+export const tinySkeletonItemSmbTheme = {
+  'ti-skeleton-item-border-radius': '2px',
+  'ti-skeleton-item-bg-color': 'var(--ti-common-color-bg-normal)',
+  'ti-skeleton-item-image-icon-color': 'var(--ti-common-color-bg-normal)',
+  'ti-skeleton-item-linear-gradient': 'linear-gradient(to right, #FAFAFA, #EBEBEB, #FAFAFA)'
```
Comment (zzcr) [inline packages/theme/src/skeleton-item/smb-theme.js]: 色号保持统一，统一用小写字母

Comment (James-9696) [reply packages/theme/src/skeleton-item/smb-theme.js]: 已修改

**Review by zzcr — APPROVED** [review]

**Pull request merged**